sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### Features Added
+
```

Comment:
Should at least have `- Initial release` or something for at least one entry ready when released.